spec/models/team_member_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe TeamMember, type: :model do
+  it "should not be able to assign team ownership to users and delete it" do
+    team = FactoryBot.create :team
```

Comment:
You could factor out this and the following line (that is currently duplicated in every `it … do` block in this file) by using something like:

```ruby
RSpec.describe TeamMember, type: :model do
  describe 'when logged in' do
    let(:user){ FactoryBot.create :user }
    let(:team){ FactoryBot.create :team }

    it "should not be able to assign team ownership to users and delete it" do
      ability = Ability.new(user)
      assert ability.cannot?(:assign_ownership, team)
    end
    …
```